Encrypt-S/kauri-wallet
# Monorepo setup

This PR restructure the project for as a monorepo, updates the travis build, add lerna for managing the repo, and updates the readme.

### Testing
- Checkout branch
- In the root dir run 'npm run lerna:bootstrap' (this installed npm modules + links repos)
- go to the kauri-api folder and start up app as normal
- confirm that everything runs normal, and you see the text `This text exported from temp-test-module` which is imported from another local package.
